chess/svg.py

```diff
@@ -162,5 +173,17 @@ def board(board=None, squares=None, flipped=False, coordinates=True, lastmove=No
             builder.append(_text(rank_name, 0, y, margin, square_size))
             builder.append(_text(rank_name, margin + 8 * square_size, y, margin, square_size))
 
+    for arrow in arrows:
+        head_file = chess.file_index(arrow.head)
+        head_rank = chess.rank_index(arrow.head)
+        tail_file = chess.file_index(arrow.tail)
+        tail_rank = chess.rank_index(arrow.tail)
+
+        xhead = (head_file + 1 if not flipped else 8 - head_file) * square_size
+        yhead = (8 - head_rank if not flipped else head_rank + 1) * square_size
+        xtail = (tail_file + 1 if not flipped else 8 - tail_file) * square_size
+        ytail = (8 - tail_rank if not flipped else tail_rank + 1) * square_size
+        builder.append("""<line x1="%f" y1="%f" x2="%f" y2="%f" stroke="#888" stroke-width="4" marker-end="url(#arrowhead)"/>""" % (xtail, ytail, xhead, yhead))
```

Comment:
![image](https://cloud.githubusercontent.com/assets/402777/23832220/f71c3000-0730-11e7-8081-a200057c80b2.png)

A couple points here:

* The arrow seems slightly off center.
* The arrow head is a bit flat.
* The arrow width does not scale with the board size.

In terms of what do aim for, I very much like the much wider (but transparent) arrows of lichess:

![image](https://cloud.githubusercontent.com/assets/402777/23832273/cbe4bc62-0731-11e7-9ae3-6e99b710363f.png)